diagnostic.sql

```diff
@@ -5,19 +5,33 @@ CREATE DATABASE "sql-diagnostic";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+surname TEXT,
```

Comment:
Don't forget the `id serial PRIMARY KEY` part. Although the database will create an ID for you, you still need to tell it to do so